src/rotator_library/anthropic_compat/streaming_fast.py

```
@@ -0,0 +1,460 @@
+
+                # Accumulate arguments
+                func = tc.get("function", {})
+                if func.get("name"):
+                    tool_calls_by_index[tc_index]["name"] = func["name"]
+                if func.get("arguments"):
+                    tool_calls_by_index[tc_index]["arguments"] += func["arguments"]
+                    yield _make_input_json_delta_event(
+                        tool_block_indices[tc_index], func["arguments"]
+                    )
+
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)